source/code/python/authentication/mfa_add_sms_factor_req.py

```diff
@@ -0,0 +1,5 @@
+sms_factor = account.factors.create({
+    'phone': {'number': '+12223334444'},
+    'type': 'SMS'},
```

Comment:
Can we also get the closing curly brace here moved down? This way it'll look a bit nicer, eg:

```python
sms_factor = account.factors.create({
    'phone': {'number': '+12223334444'},
    'type': 'SMS',
}, challenge=False)